winch/codegen/src/isa/x64/masm.rs

```diff
@@ -98,6 +88,70 @@ impl Masm for MacroAssembler {
         self.asm.trapif(IntCmpKind::GtU, TrapCode::StackOverflow);
     }
 
+    fn save_clobbers(&mut self, clobbers: &[(Reg, OperandSize)]) {
+        // Determine how much space we need for the clobbers
+        let clobbered_size = align_to(
+            clobbers
+                .iter()
+                .fold(0u32, |total, (reg, _)| match reg.class() {
+                    RegClass::Int => total + 8,
+                    RegClass::Float => align_to(total, 16) + 16,
```

Comment:
Would it make sense to use `ABI::word_bytes()` and `ABI::word_bytes() * 2` here?

Reply:
I like that! I think it also gives good justification to uncommenting the debug assert you mentioned below, as now it's checking the connection between the `word_bytes()` and `OperandSize::bytes()` results.